docs-2.0/20.appendix/release-notes/dashboard-comm-release-note.md

```diff
@@ -1,12 +1,67 @@
+  - Properties of vertices or edges [#5230](https://github.com/vesoft-inc/nebula/pull/5230) [#4846](https://github.com/vesoft-inc/nebula/pull/4846) [#4841](https://github.com/vesoft-inc/nebula/pull/4841) [#5238](https://github.com/vesoft-inc/nebula/pull/5238)
+  - Functions and aggregations [#5135](https://github.com/vesoft-inc/nebula/pull/5135) [#5121](https://github.com/vesoft-inc/nebula/pull/5121) [#4884](https://github.com/vesoft-inc/nebula/pull/4884)
+  - Using illegal data types [#5242](https://github.com/vesoft-inc/nebula/pull/5242)
+  - Clauses and operators [#5241](https://github.com/vesoft-inc/nebula/pull/5241) [#4965](https://github.com/vesoft-inc/nebula/pull/4965)
+
+- Fix the bugs related to DDL and DML statements:
+
+  - ALTER TAG [#5105](https://github.com/vesoft-inc/nebula/pull/5105) [#5136](https://github.com/vesoft-inc/nebula/pull/5136)
+  - UPDATE [#4933](https://github.com/vesoft-inc/nebula/pull/4933)
+
+- Fix the bugs related to other functions:
+
+  - TTL [#4961](https://github.com/vesoft-inc/nebula/pull/4961)
+  - Authentication [#4885](https://github.com/vesoft-inc/nebula/pull/4885)
+  - Services [#4896](https://github.com/vesoft-inc/nebula/pull/4896)
+
+## Change
+
+- The added property name can not be the same as an existing or deleted property name, otherwise, the operation of adding a property fails. [#5130](https://github.com/vesoft-inc/nebula/pull/5130)
+- Limit the type conversion when modifying the schema. [#5098](https://github.com/vesoft-inc/nebula/pull/5098)
+- The default value must be specified when creating a property of type `NOT NULL`. [#5105](https://github.com/vesoft-inc/nebula/pull/5105)
+- Add the multithreaded query parameter `query_concurrently` to  the configuration file with a default value of `true`. [#5119](https://github.com/vesoft-inc/nebula/pull/5119)
+- Remove the parameter `kv_separation` of the KV separation storage function from the configuration file, which is turned off by default. [#5119](https://github.com/vesoft-inc/nebula/pull/5119)
+- Modify the default value of `local_config` in the configuration file to `true`. [#5119](https://github.com/vesoft-inc/nebula/pull/5119)
+- Consistent use of `v.tag.property` to get property values, because it is necessary to specify the Tag. Using `v.property` to access the property of a Tag on `v` was incorrectly allowed in the previous version. [#5230](https://github.com/vesoft-inc/nebula/pull/5230)
+- Remove the column `HTTP port` from the command `SHOW HOSTS`. [#5056](https://github.com/vesoft-inc/nebula/pull/5056)
+- Disable the queries of the form `OPTIONAL MATCH <pattern> WHERE <condition>`. [#5273](https://github.com/vesoft-inc/nebula/pull/5273)
+- Disable TOSS. [#5119](https://github.com/vesoft-inc/nebula/pull/5119)
+- Rename Listener's pid filename and log directory name. [#5119](https://github.com/vesoft-inc/nebula/pull/5119)
+
+## Legacy versions
+
+[Release notes of legacy versions](https://nebula-graph.io/posts/)
```

Comment:
This link does not directly lead to the Release notes of the legacy versions page.